scripts/linux/buildAPIProxy.sh

```diff
@@ -151,11 +151,11 @@ build_project()
 {
     # build project with cross
     if [[ "$ARCH" == "amd64" ]]; then
-        execute scripts/linux/cross-platform-rust-build.sh --os ubuntu18.04 --arch "amd64" --build-path edge-modules/api-proxy-module
+        execute scripts/linux/cross-platform-rust-build.sh --os alpine1.19.3 --arch "amd64" --build-path edge-modules/api-proxy-module
```

Comment:
Just curious, why do we need execute?

Reply:
We need to build the API proxy executable. API proxy consists of both nginx proxy and a rust executable (that monitors twins, get certs from worload API etc...) . Here we build the rust executable. The target is alpine, hence the change.

Reply:
Oh sorry I was asking about the execute keyword. It doesn't matter though.